lib/createsend/client.py

```diff
@@ -29,9 +29,17 @@ def details(self):
         response = self._get("/clients/%s.json" % self.client_id)
         return json_to_py(response)
 
-    def campaigns(self):
+    def campaigns(self, sent_from_date="", sent_to_date="", tags="", page=1, page_size=1000, order_direction="desc"):
```

Comment:
All of the new parameters have default value, so they could call it like before (although the object returns now different)